tests/integration/sync/syncModel.test.ts

```diff
@@ -99,10 +102,23 @@ describe.concurrent("Sync environment from folder", () => {
       .then(stats => stats.isFile())
       .catch(() => false);
 
+    const webSpotlightJsonExists = await fsPromises.stat(`${folderPath}/${webSpotlightFileName}`)
```

Comment:
Thank you, I didn't notice this when adding the entities. Can we make the check automatically for all the constants in the file?